src/modules/playRoom/components/CountDown.js

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState }  from 'react';
+
+const CountDown = () => {
+  const [countDown, setCountDown] = useState(59);
+  useEffect(() => {
+    const time = setInterval(() => {
+      setCountDown(countDown - 1);
+    }, 1000);
+    if (countDown === -1) {
```

Comment:
because when the countdown goes to -1, on the display we will see from 0 jump to 59. If I set countdown is 0, on the display we will see from 1 jump to 59

Reply:
I mean why you have to reset ```countDown``` value to 59, why you don't hide count down and start calculate result

Reply:
Oh, I didn't think of that. I will resolve it. Thank you.